gamestonk_terminal/stocks/sector_industry_analysis/sia_controller.py

```diff
@@ -724,6 +794,121 @@ def call_metric(self, other_args: List[str]):
                 self.stocks_data,
             )
 
+    def call_satma(self, other_args: List[str]):
+        """Process satma command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="satma",
+            description="See all metrics available",
+        )
+
+        parser.add_argument(
+            "-s",
+            "--statement",
+            dest="statement",
+            help="See all metrics available for the given choice",
+            choices=["BS", "bs", "IS", "is", "CF", "cf"],
+        )
+
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+
+        help_text = ""
+        statement_string = {
+            "BS": "Balance Sheet Statement \n",
+            "IS": "Income Statement \n",
+            "CF": "Cash Flow Statement \n",
+        }
+
+        if ns_parser.statement:
+            ns_parser.statement = str(ns_parser.statement).upper()
+            if ns_parser.statement == "BS":
```

Comment:
If you take one step closer here you notice that the code is very much the same. This is because you compare on the if condition but then created this logic where you actually don't need an if/else. This can be simply:

```
if ns_parser.statement:
   ns_parser.statement = str(ns_parser.statement).upper()

  if ns_parser.statement not in ("IS", "BS", "CF"):
     console.print("This type of information doesn't exist").

   help_text += statement_string[ns_parser.statement]
   for k, v in stockanalysis_model.sa_keys[ns_parser.statement].items():
       help_text += f"  {k} {(20 - len(k)) * ' '} {v} \n"

else:
   for statement, statement_value in stockanalysis_model.sa_keys.items():
      help_text += statement_string[statement]
      for k, v in statement_value.items():
          help_text += f"  {k} {(20 - len(k)) * ' '} {v} \n"
```

Reply:
Agreed that this is a better format!